apps/bare-expo/package.json

```diff
@@ -80,6 +80,7 @@
     "@babel/runtime": "^7.5.5",
     "@react-navigation/web": "2.0.0-alpha.0",
     "expo": "~37.0.0",
+    "expo-image": "~1.0.0",
```

Comment:
Just an idea, but maybe start off from some minor version like `0.1.0` 🤔 😄

Reply:
:+1:, I will go with `1.0.0-alpha.0` since that's what ~kzzzf used when creating `react-native-screens`:

https://github.com/software-mansion/react-native-screens/commit/31281d6b68c402bbb88fc3e2f1d7cfc05e8fb170#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R3